packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,479 @@
+| overflowIndex     | number                                  | 1             | Index of item where BreadcrumbMenu should be shown                                       |
+| size              | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb                                                           |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button
+- Link
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem onClick={() => console.log('smth...')}>
+  Item 2
+</BreadcrumbItem>
+<BreadcrumbItem href="#">
+  Item 3
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default   | Purpose                                     |
+| ------------ | --------------------------------------- | --------- | ------------------------------------------- |
+| active       | boolean                                 | false     | Indicates if the item is active             |
+| disabled     | boolean                                 | false     | Disables Breadcrumb item                    |
+| divider      | _slot_                                  | `chevron` | Sets type of divider                        |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`     | Sets the focus behavior for the Breadcrumb. |
+| icon         | _slot_                                  |           | Sets icon                                   |
+| iconPosition | `before`, `after`                       | `before`  | Sets icon position                          |
+| tooltip      | _slot_                                  | undefined | Sets custom tooltip                         |
+
+#### Breadcrumb icon
+
+An icon can be SVG or image.
+The icon might be added as a prop:
+
+```jsx
+<BreadcrumbItem icon={<IconComponent />}>Item 1</BreadcrumbItem>
+<BreadcrumbItem icon="../path/name.png">Item 1</BreadcrumbItem>
+<BreadcrumbItem>
+  <SomeSVG />
+  Item 1
+</BreadcrumbItem>
+```
+
+### BreadcrumbDivider
+
+#### DOM
+
+```HTML
+<li aria-hidden="true">
+  {children}
+</li>
+```
+
+#### API
+
+| Property | Values                               | Default   | Purpose              |
+| -------- | ------------------------------------ | --------- | -------------------- |
+| divider  | `chevron`, `slash`, custom component | `chevron` | Sets type of divider |
+
+### BreadcrumbMenu
+
+Dropdown menu is used for collapsed items and subfolders.
+BreadcrumbMenu uses @fluentui/react-menu component.
+
+### BreadcrumbLink
```

Comment:
Does this mean a developer could use both implementations?
```tsx
<BreadcrumbItem href="#">Link</BreadcrumbItem>
```
```tsx
<BreadcrumbItem>
  <BreadcrumbLink href="#">Link</BreadcrumbLink>
</BreadcrumbItem>
```

Reply:
I'm also wondering about this, I can't see any usage examples of a lot of the components

Reply:
@marcosmoura, that's why I wrote the question above. I'm not sure if a Link should be a separate component. I think it can be a slot. But how will it work with tree-shaking? Maybe for this case, it's good to have a Button and Link as a separate components. That's what I wanted to discuss
@ling1726 there were no usage examples because it was a first version of a spec which is 2 weeks on review by now. I'm gonna update it today with more examples and updates. This PR was for early feedback, but it didn't work. Looks like I'll need to remake some things.